### ISSUE TITLE
`&backend` equivalent for basic types

### ISSUE THREAD

**Issue by ynadji** [zeek/zeek#2306]
I had a recent use case where I needed to update the state of a boolean flag across the cluster to change behavior on workers and proxies. I initially was using `Broker::publish` to the `Cluster::{worker,proxy,manager}_topic` until @awelzel helpfully suggested having the variable be an `option` and using `Config::set_value` to make and propagate the update. This works, however, it seems it would be nicer to have something similar without having to use `set_value` and without exposing it as a config option.

Being able to do something like:

```ruby
global flag = F &clusterized;

...

event blahblah()
    {
    flag = T;
    }
```

and have the change to `flag` propagate would be very nifty, alleviate boilerplate code, and fit the use case better than abusing an `option`. I would've preferred the attribute to be `&synchronized`, but IIRC that keyword was used earlier and I wouldn't want to cause confusion. The name is just a suggestion.

**Comment by awelzel** [comment]
> until @awelzel helpfully suggested having the variable be an option and using Config::set_value to make and propagate the update

Yeah, if the variable semantically isn't an option, it's not great and I'd be curious to hear other ideas (at least your `Site::local_nets` use-case fell under the option umbrella though).

**Comment by ynadji** [comment]
Just to add, it appears that [according to the docs](https://docs.zeek.org/en/master/script-reference/attributes.html#attr-&backend) only tables/sets support `&backend`. It would be nice to either (1) include `vector`s in this eventual solution or (2) add support for `vector`s in `&backend`.

**Comment by ckreibich** [comment]
I agree that this would be nice to have, but the whole implementation down through Broker is currently based on a key/val paradigm, so this would be quite a bit of work. Seems you could work around this by putting the needed types into a table — is there a technical reason why this wouldn't work?

**Comment by ynadji** [comment]
Yeah, that's a fine work around it's just clunky. I've also defined variables as `option`s and used the configuration framework to keep the value synced across the cluster, which is similarly clunky. I'm not immediately blocked by anything here, but thought it would be a nice feature to have.

**Comment by timwoj** [comment]
`&backend` is deprecated as of https://github.com/zeek/zeek/pull/5010 and will be removed in a later version of Zeek. This issue can likely be closed.
